### ISSUE TITLE
Syntax error, script broken

### ISSUE THREAD

**Issue by AlyoshaVasilieva** [mouseas/steamSummerMinigame#97]
From Chrome console:

```
Syntax error @ 'Monster Minigame AutoScript'!
##########################
JSHINT output:
script: (W033) Missing semicolon on line: 84 at character: 2
script: (W033) Missing semicolon on line: 103 at character: 10
script: (W033) Missing semicolon on line: 177 at character: 6
script: (W004) 'i' is already defined on line: 255 at character: 20
script: (W041) Use '===' to compare with '0' on line: 264 at character: 37
script: (W033) Missing semicolon on line: 366 at character: 6
script: (W041) Use '!==' to compare with 'undefined' on line: 381 at character: 39
script: (W041) Use '!==' to compare with 'undefined' on line: 383 at character: 71
script: (W038) 'upgradeCost' used out of scope on line: 439 at character: 9
script: (W038) 'upgradeCost' used out of scope on line: 443 at character: 76
script: (W038) 'upgradeCost' used out of scope on line: 456 at character: 9
script: (W038) 'upgradeCost' used out of scope on line: 463 at character: 21
script: (W038) 'upgradeCost' used out of scope on line: 475 at character: 5
script: (W038) 'upgradeCost' used out of scope on line: 477 at character: 17
script: (W032) Unnecessary semicolon on line: 502 at character: 2
script: (W041) Use '===' to compare with '0' on line: 538 at character: 39
script: (W041) Use '===' to compare with '0' on line: 575 at character: 39
script: (W116) Expected '(end)' and instead saw '}' on line: 595 at character: 1
##########################

SyntaxError: Missing catch or finally after try
    at Function (native)
    at d (eval at <anonymous> (unknown source), <anonymous>:27:37)
    at eval (eval at <anonymous> (eval at <anonymous> (unknown source)), <anonymous>:1:23)
    at eval (native)
    at m (eval at <anonymous> (unknown source), <anonymous>:53:418)
    at Z (eval at <anonymous> (unknown source), <anonymous>:27:80)
    at aa (eval at <anonymous> (unknown source), <anonymous>:63:497)
    at b (eval at <anonymous> (unknown source), <anonymous>:64:398)
    at v.push.fn (eval at <anonymous> (unknown source), <anonymous>:32:209)(anonymous function) @ (program):28
```

The script is not changing lanes, so I do not think it is working. Using version ffc61ef48fe6aa89537f3da77c09ffc8950615a9.

**Comment by win0na** [comment]
I pushed a fix at https://github.com/mouseas/steamSummerMinigame/commit/d77a05f9df8574f7a9a2d7d92b762fe896deea5f. Would you mind testing, please?

**Comment by ghost** [comment]
Just tested it, I get `Uncaught ReferenceError: purchaseUpgradesToggle is not defined` as it is only mentioned in one place.

**Comment by ghost** [comment]
Ok, so whoever did the most recent toggle didn't test it. you have `purcahseUpgradeToggle`, and `purchaseUpgradesToggle`

**Comment by win0na** [comment]
That was me, and it was a slight typo error on my part. I've fixed it in https://github.com/mouseas/steamSummerMinigame/commit/ad1ec42092baadf70aae4e8533187c33354eb17f.

**Comment by ghost** [comment]
Not yet, one says purchase, the other purcauhase

**Comment by win0na** [comment]
_facepalm_ Fixed in https://github.com/mouseas/steamSummerMinigame/commit/416e6d21b7b4ae776f72795ee29af48f49f79869.

**Comment by ghost** [comment]
haha ok now I'll test it. We all have our moments

**Comment by win0na** [comment]
Okay, I swear this isn't intentional. I've _finally_ fixed it in https://github.com/mouseas/steamSummerMinigame/commit/76c49ab9915ef388a667aad5fca814016c572a6c.

**Comment by ghost** [comment]
hahahahaha the spelling gods don't like you today my friend

**Comment by win0na** [comment]
![b3tmibk](https://cloud.githubusercontent.com/assets/4269491/8168701/4215f74c-136a-11e5-89c6-80f8170743c3.png)

**Comment by ghost** [comment]
I'm still getting some errors: `Uncaught TypeError: Cannot read property 'childElements' of null`

**Comment by win0na** [comment]
Could you attempt to fix it for about an hour? I just left my house.

**Comment by ghost** [comment]
Yeah no problem, I'm on it

**Comment by ghost** [comment]
Don't know what happened, but it seems to have fixed it self ¯\_(ツ)_/¯

**Comment by IWannaBeATiger** [comment]
Something seems to be wrong with the cooldown on tactical nuke whenever it's off CD when it tries to use it

Tactical Nuke is purchased, cooled down, and needed. Nuke 'em.
VM2408:665 

Uncaught ReferenceError: getCooldownTime is not defined

**Comment by ghost** [comment]
I'm fixing it now

**Comment by ghost** [comment]
It would seem commit https://github.com/mouseas/steamSummerMinigame/commit/ee013ebd4f82f56da5b78c9d1a507709beb7a1b6 got rid of a bunch of functions: `updateAvgClickRate()`, `getCooldownTime()`, `disableFlinchingAnimation()`, `disableDamageText()`

**Comment by ZedTheYeti** [comment]
@JCKE I just merged Pull Request #99 in which added those back.

**Comment by ghost** [comment]
Perfect, this issue can probably be closed then.
